kinetic/distribution.yaml

```diff
@@ -13251,6 +13251,21 @@ repositories:
       url: https://github.com/tork-a/roswww.git
       version: develop
     status: maintained
+  rowma_ros:
+    doc:
+      type: git
+      url: https://github.com/rowma/rowma_ros.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/rowma/rowma_ros.git
+      version: 0.0.2-2
+    source:
+      type: git
+      url: https://github.com/rowma/rowma_ros.git
```

Comment:
The source repository here has a LICENSE file for GPLv3.  But the package.xml says "MIT".  Can you please update one of those to clarify which license this project is under?